.config/nextest.toml

```diff
@@ -0,0 +1,3 @@
+[profile.ci]
+failure-output = "immediate-final"
+fail-fast = false
```

Comment:
Could we override where we have to put this file so we can keep it contained in the GitHub Workflows directory?